atuin-client/src/sync.rs

```diff
@@ -107,6 +107,8 @@ async fn sync_upload(
 
         for i in last {
             let data = encrypt(&i, &key)?;
+            let data = serde_json::to_string(&data)?;
```

Comment:
`let data` twice??!

Reply:
Yup! Imo it's cleaner to read that way, and is something Rust allows you to do :)